runelite-client/src/main/java/net/runelite/client/plugins/party/PartyPlugin.java

```diff
@@ -435,6 +516,9 @@ public void onUserPart(final UserPart event)
 			}
 
 			worldMapManager.remove(removed.getWorldMapPoint());
+
+			panel.updateParty();
```

Comment:
Shouldn't this be updating the party after removing the member?

Reply:
Doesn't matter, they affect the UI based on the PartyService list.